lib/errors/arsenalErrors.ts

```diff
@@ -365,6 +365,11 @@ export const NoSuchWebsiteConfiguration: ErrorFormat = {
     description: 'The specified bucket does not have a website configuration',
 };
 
+export const NoSuchTagSet: ErrorFormat = {
+    code: 404,
+    description: 'The TagSet does not exist',
```

Comment:
Why not:

```suggestion
    description: 'There is no tag set associated with the bucket.',
```

To be standard with the AWS error description? :smile:

Reply:
As mentioned in PM, the new message is 'The TagSet does not exist'
`aws s3api get-bucket-tagging --bucket anuragplatform

An error occurred (NoSuchTagSet) when calling the GetBucketTagging operation: The TagSet does not exist`